react/__tests__/Arrows.test.tsx

```diff
@@ -0,0 +1,144 @@
+import React, { Fragment } from 'react'
+import { render, fireEvent } from '@vtex/test-tools/react'
+
+import { useSliderState } from '../components/SliderContext'
+import Arrow from '../components/Arrow'
+
+const mockGoForward = jest.fn()
+const mockGoBack = jest.fn()
+
+jest.mock('../components/SliderContext', () => ({
+  useSliderState: jest.fn(),
+}))
+
+jest.mock('../hooks/useSliderControls', () => ({
+  useSliderControls: () => ({
+    goForward: mockGoForward,
+    goBack: mockGoBack,
+  }),
+}))
+
+beforeEach(() => {
+  mockGoForward.mockClear()
+  mockGoBack.mockClear()
+  // eslint-disable-next-line prettier/prettier
+})
+
+describe('Accessibility', () => {
+  ;(useSliderState as jest.Mock).mockImplementation(() => ({
```

Comment:
Nit, I prefer to do something like:

```tsx
const mockUseSliderState = useSliderState as jest.Mock<SliderState>

mockUseSliderState.mockImplementation(
```

Then I won't have to repeat this typing

Reply:
Yeah! That's much better.
I changed to this solution in some tests, but I guess I forgot to do it in this test file 😅